docs/security-features/network/open-ports.rst

```diff
@@ -1,11 +1,130 @@
 No Open Ports
+=============
+
+A port bound to a service that is actively listening for incoming packets on 
+a network is referred to as an "open port". Exposing open ports on untrusted 
+networks can pose a security risk if the listening service has a vulnerability 
+or is misconfigured.
+
+Starting with Ubuntu 6.06 LTS Dapper Drake, Ubuntu has adhered to a "No Open Ports" policy, which 
+states that, besides certain exceptions, default installations of Ubuntu must 
+have no listening network services after initial install.
+
+Guidelines for Default Network Services
++++++++++++++++++++++++++++++++++++++++
+
+While the network services exposed on default Ubuntu installations are kept to 
+a minimum, some exceptions have been permitted. These are generally limited only
+to services that provide substantial improvement to out-of-the-box user experience.
+
+All such network services must adhere to the following guidelines:
+
+Exposing local information to the network
+-----------------------------------------
+
```

Comment:
Note: Usually we need an introductory sentence for a list. We can probably make an exception here, since they all are policies/requirements of the same guidelines, separated by headings. That's an unusual design. Make sure it looks alright rendered.